src/AmazonSES.php

```diff
@@ -134,7 +134,7 @@ public function setSignatureVersion($signatureVersion)
      * @param bool $verifyHost
      * @param LoggerInterface|null $logger
      */
-    public function __construct($accessKey = null, $secretKey = null, $host = self::AWS_US_EAST_1, $verifyPeer = true, $verifyHost = true, LoggerInterface $logger = null, string $signatureVersion = SimpleEmailService::REQUEST_SIGNATURE_V4)
+    public function __construct($accessKey = null, $secretKey = null, $host = self::AWS_US_EAST_1, $verifyPeer = true, $verifyHost = true, LoggerInterface $logger = null, string $signatureVersion = 'v4')
```

Comment:
I was talking about [scalar type](https://www.php.net/manual/en/language.types.intro.php) hinting.

Can you please remove `string` in front of argument?

```
public function __construct(. . . . . . .  string $signatureVersion = 'v4')
{
}
```

I think it should be like

```
public function __construct(. . . . . . .  $signatureVersion = SimpleEmailService::REQUEST_SIGNATURE_V4)           // <----- removed string as string is scalar type
{
}
```

For **constant** I think you can keep the constant as it was before. 

- Also it would be great if phpdoc block is updated.